roles/edpm_nova/tasks/configure.yml

```diff
@@ -101,6 +102,32 @@
   tags:
     - edpm_users
 
+- name: Sync nova config files
+  become: true
+  tags:
+    - configure
+    - nova
+  ansible.posix.synchronize:
+    src: "{{ edpm_nova_config_sync_dest }}"
+    dest: "{{ edpm_nova_config_dest }}"
+    recursive: true
+    delete: true
```

Comment:
OK, this is the meat of the patch, we want to make sure that the files that are not provided by the user are deleted

Reply:
ya so synchonize uses rsync and adding delete with recusitive is basically like passing mirror

with this current config it will copy all files form the source that are newer then on the test and delete any files on the dest not found on the source.